.github/workflows/bigtable-emulator-system-tests.yaml

```diff
@@ -17,7 +19,7 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
-      - run: ./.github/emulator/start-emulator.sh bigtable
+      - run: ./.github/emulator/start-emulator.sh bigtable 419.0.0-emulators
```

Comment:
nit: Can we export these as variables in the workflow?

Reply:
Exposed this as `EMULATOR_VERSION` in the script instead which adds a default.

Adding as env vars in workflow, but not in the run cmd was adding confusion as script was triggered without providing a version but it's use appeared magically inside the script.